README.md

```diff
@@ -11,40 +11,19 @@ Service Manual Publisher is used for publishing and maintaining https://www.gov.
 
 ## Technical documentation
 
-PostgreSQL-backed Rails publishing application for internal use, with no public facing aspect.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
 There is some [disparity between the content as it appears in the database, and the content as it appears in the publishing api](doc/arch/002-disparity-between-database-and-publishing-api.md). This will need to be addressed if we switch to using the Publishing API as our main data store in the future.
 
 The Service manual's guide pages have a nested URL structure which is unusual for GOV.UK. The pros and cons to the nested URL structure are [retrospectively documented](doc/arch/001-nested-url-structure.md).
 
-### Development
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
```

Comment:
We normally add a final line here: "Use GOV.UK Docker to run any commands that follow."

Reply:
I've amended the commit in dc2bafa to fix this.